js/fc.js

```diff
@@ -296,9 +359,9 @@ var FC = {
 
         RX_CONFIG = {
             serialrx_provider:      0,
-            maxcheck:               0,
-            midrc:                  0,
-            mincheck:               0,
+            stick_max:               0,
+            stick_center:                  0,
```

Comment:
there will be many spaces, i did LOTS of search and replacing today.  I'll go over it again after it's functional, no point aligning things at this stage.